app/controllers/errors_controller.rb

```diff
@@ -0,0 +1,5 @@
+class ErrorsController < ApplicationController
+	def error404
+		render template: "errors/simple_404.html.erb", status: :not_found
+	end
+end
```

Comment:
@OrthoDex for this project we use 2 spaces for the formatting files and we have a newline on the end.

Also what do you think about this?

```ruby
class ErrorsController < ApplicationController
  def not_found
    render template: "errors/not_found.html.erb", status: :not_found
  end
end
```